itensor/iterativesolvers.h

```diff
@@ -937,6 +953,245 @@ arnoldi(const BigMatrixT& A,
     return res;
     }
 
+template <class BigMatrixT>
+Real
+krylov(BigMatrixT const& A,
+	   ITensor& phi,
+	   Real t,
+	   Args const& args)
+	{
+    auto maxiter_ = args.getSizeT("MaxIter",30);
+    auto errgoal_ = args.getReal("ErrGoal",1E-10);
+    auto debug_level_ = args.getInt("DebugLevel",-1);
+	int ideg_ = args.getInt("Ideg",6);
```

Comment:
Could we make this argument name more descriptive, like `PadeApproxDeg`?

Reply:
Good suggestion. I have changed that.